tools/integration-tests/tester/tests/testutil.go

```diff
@@ -7,6 +7,8 @@ import (
 	"testing"
 	"time"
 
+	"github.com/iotaledger/hive.go/crypto/ed25519"
```

Comment:
Moved it

However, I noticed that the dog didn't bark, and when I do `goimports -w tester/tests/consensus/consensus_conflict_spam_test.go` or "optimize imports" command it doesn't move automatically...

I don't see a style-guide in the repo... so currently I just moved to where you think you wanted me to move it. Since `goimports` also accepts it I think I did fine.

Back when I was reviewing code all I cared is that the imports are in the order compatible with everyone's IDE settings. So that people won't reorganize imports on every commit and I could easily see changes in dependencies. 

So is there some setting I am missing. I just believe that import order is should be something purely automatic and no one should spend time commenting about it or  fixing it like I am doing now :-)